stat/distuv/interfaces.go

```diff
@@ -4,19 +4,23 @@
 
 package distuv
 
+// LogProber calculates the log-probability for a given value of a random variable.
 type LogProber interface {
 	LogProb(float64) float64
```

Comment:
```
// LogProb returns the natural logarithm of the
// value of the probability density function at x.
LogProb(x float64) float64
```

Reply:
What about discrete distributions, which do not have the PDF but a probability mass function?

Reply:
```
// LogProb returns the natural logarithm of the
// value of the probability density or probability
// mass function at x.
LogProb(x float64) float64
```
?

Presumably equivalent text in `Prober`.

Reply:
Done. Where's `Prober`?

Reply:
Doesn't exist yet I suppose. We don't need it if it's not there.